doc/sphinx-guides/source/admin/external-tools.rst

```diff
@@ -94,6 +94,14 @@ File level preview tools allow the user to see a preview of the file contents wi
 
 When a file has a preview available, a preview icon will appear next to that file in the file listing on the dataset page. On the file page itself, the preview will appear in a Preview tab either immediately or once a guestbook has been filled in or terms, if any, have been agreed to.
 
+File Level Query Tools
+++++++++++++++++++++++++
+
+File level query tools allow the user to ask natural language queries of data table's contents without having to download it.
+
+When a file has a query tool available, a query icon will appear next to that file in the file listing on the dataset page. On the file page itself, the query tool will appear in a Query tab either immediately or once a guestbook has been filled in or terms, if any, have been agreed to.
```

Comment:
If you have the spreadsheet previewer enabled, Ask Dataverse is showing up under the Preview tab, not the Query tab. Is this a bug? Or by design? If so we should mention that you might need to look for it under the Preview tab. Here's a screenshot:

![Screen Shot 2023-07-26 at 2 20 26 PM](https://github.com/IQSS/dataverse/assets/21006/90a2c0e4-1a77-4012-830f-c4cde99f02ea)

Reply:
Updated so that if there are multiple tools available, the tab will be labeled "File Tools" if there's only one then we will show either "Preview" or "Query" as applicable